### ISSUE TITLE
Learning rate scheduling

### ISSUE THREAD

**Issue by Jayanth-Balasubramanian** [Jayanth-Balasubramanian/Pneumonia-Detection#3]
### Problem:
Adam optimizer's learning rate is set to its default (0.001)

### Solution: 
Although Adam does maintain adaptive gradients per parameter, a learning rate scheduler could help change the learning rate if a performance metric's improvement has slowed, for example. Explore the use of an LR scheduler with validation accuracy as the metric.
